hibernate-core/src/main/java/org/hibernate/dialect/Dialect.java

```diff
@@ -1573,7 +1579,18 @@ public String getCreateMultisetTableString() {
 	public SqmMultiTableMutationStrategy getFallbackSqmMutationStrategy(
 			EntityMappingType entityDescriptor,
 			RuntimeModelCreationContext runtimeModelCreationContext) {
-		throw new NotYetImplementedFor6Exception( getClass() );
+		if ( entityDescriptor.getIdentifierMapping() instanceof CompositeIdentifierMapping) {
+			if ( !supportsTuplesInSubqueries() ) {
+				return new InlineStrategy( this );
+			}
+		}
+		
+		return new PersistentTableStrategy(
+				new IdTable( entityDescriptor, name -> name ),
+				AfterUseAction.CLEAN,
+				PhysicalIdTableExporter::new,
```

Comment:
Is this right?

Reply:
Yes, we want to export the physical table

Reply:
Thanks.